lib/locreator.js

```diff
@@ -75,6 +75,12 @@ Locreator.prototype.addStarMethods = function locreator(locatorId, _locator, par
   locatorObject[locatorId + 'OptionValue'] = function (optionValue) {
     return drivex.selectByOptionValue(locator, optionValue, parentWebElement);
   };
+  locatorObject[locatorId + 'ValidateText'] = function (ValidateText) {
```

Comment:
What about calling the method "TextEquals" instead? Trying to shorten the name if possible